### PR TITLE
Update delegated access option and freshness pass

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by msewaweru** [comment]
@merill, please review this update.

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 1]
```diff
@@ -39,11 +39,31 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
     Connect-MgGraph -AccessToken $AccessToken
     ```
 
+#### Use delegated access using own App Id
+
+To follow the least privilege model for the applications in your Azure AD tenant, you might be concerned about consenting to many permissions scopes to the Microsoft Graph PowerShell app over time. To avoid this, you can register your own app for use with Microsoft Graph PowerShell. This allows you to have more granular control.
```
Comment (merill) [inline microsoftgraph/docs-conceptual/authentication-commands.md:44]: I would rephrase this:
To follow the least privilege model for the applications in your Azure AD tenant, you might be concerned about consenting to many permissions scopes to the Microsoft Graph PowerShell app over time. To avoid this, you can register your own app for use with Microsoft Graph PowerShell. This allows you to have more granular control.

Follow the steps below to set up your own app:

To:
Follow the steps below to create custom applications that can be used to connect to Microsoft Graph PowerShell. Use this approach if you need to limit the consent permissions granted for Graph PowerShell usage.

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 2]
```diff
@@ -39,11 +39,31 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
...
+
+Follow the steps below to set up your own app:
+
+1. Browse to **Entra** > **App registrations** [adappreg.cmd.ms](https://entra.microsoft.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade/quickStartType~/null/sourceType/Microsoft_AAD_IAM/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) > **New Registration**
```
Comment (merill) [inline microsoftgraph/docs-conceptual/authentication-commands.md:48]: I would remove the reference to _adppreg.cmd.ms_

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 3]
```diff
@@ -39,11 +39,31 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
...
+    1. **Name**: *Microsoft Graph PowerShell - High Privilege admin use only* (Give a meaningful name)
```
Comment (merill) [inline microsoftgraph/docs-conceptual/authentication-commands.md:49]: I would change 'Give a meaningful name' to 'Provide a meaningful name'

File: microsoftgraph/docs-conceptual/authentication-commands.md (hunk 2 above)
Comment (msewaweru) [reply microsoftgraph/docs-conceptual/authentication-commands.md:48]: > I would remove the reference to _adppreg.cmd.ms_

I think it would be good to provide a link to the app registration page for quick access. I will however change the link name. I also forgot that we should be referring to Entra yet in docs until sometime in August. I will edit that as well.

**Comment by msewaweru** [comment]
Thank you, @Merill, for your review of this update. I have addressed your feedback and this is ready for a rereview.

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 4]
```diff
@@ -39,11 +39,29 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
     Connect-MgGraph -AccessToken $AccessToken
     ```
 
+#### Use delegated access using own App Id
```
Comment (merill) [inline microsoftgraph/docs-conceptual/authentication-commands.md:42]: Can I propose this for the heading?

Use delegated access with a custom application for Graph PowerShell

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 5]
```diff
@@ -39,11 +39,29 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
...
+
+Follow the steps below to create custom applications that you can use to connect to Microsoft Graph PowerShell. Use this approach if you need to limit the consent permissions granted for Graph PowerShell usage.
```
Comment (merill) [inline microsoftgraph/docs-conceptual/authentication-commands.md:44]: A small change here to make it easier to understand.

Follow the steps below to create custom applications that you can use to connect to Microsoft Graph PowerShell. Use this approach if you need to isolate and limit the consent permissions granted for Graph PowerShell usage.

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 6]
```diff
@@ -39,11 +39,29 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
...
+
+1. Go to the <a href="https://entra.microsoft.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade" target="_blank">Azure portal - App registrations</a> > **New Registration**
+    1. **Name**: *Microsoft Graph PowerShell - High Privilege admin use only* (Enter a meaningful name for your application)
+    1. **Supported account types**: Accounts in this organization directory
+    1. **Redirect URI**:
+        - Select **Public client/native** from the drop down
+        - **Uri**: http://localhost
+        - Click **Create**
+
```
Comment (merill) [inline microsoftgraph/docs-conceptual/authentication-commands.md:53]: @msewaweru The Create button appears to have been renamed to Register in the UI. Need to update the instruction above to say Click Register instead of Click Create.

We also need to add these steps to make the process complete.

1. Select **Overview** blade and click **Go to Enterprise applications**
1. Select the **Properties** blade and set **Assignment required?** to **Yes**
1. Click **Save**
1. Select the **Users and groups** blade and add the users and groups permitted to use this Graph PowerShell app.

**Comment by msewaweru** [comment]
@Merill- appreciate your feedback on the changes. I have addressed all the feedback provided. Thanks.

**Comment by merill** [comment]
Cheers @msewaweru LGTM

Thanks!

**Comment by msewaweru** [comment]
@timayabi2020, please merge this.

**Review by timayabi2020 — APPROVED** [review]
